ReconcileProjects/index.ts

```diff
@@ -26,7 +28,8 @@ const timerTrigger: AzureFunction = async (context: Context): Promise<void> => {
   try {
     conn = await connectionFactory(context);
 
-    await checkForNewProjects(context, projects, conn);
+    // FIXME If a project is already in the db, but one of its hardcoded properties changes, the project in the db is not updated. Use mongoose's save function to do the heavy lifting of seeing whether a project is different and changing only the new properties.
```

Comment:
Great idea, would love to fix this.

Reply:
Yup, looking forward to taking care of that after Blonks is hooked up.